content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -41,14 +41,77 @@ The following diagram illustrates the RUM event hierarchy:
+| `device.brand`  | string | The device brand as reported by the device (User-Agent HTTP header)  |
+| `device.model`   | string | The device model as reported by the device (User-Agent HTTP header)   |
+| `device.name` | string | The device name as reported by the device (User-Agent HTTP header) |
+
+### Operating system
+
+The following OS-related attributes are attached automatically to all events collected by Datadog:
+
+| Attribute name                           | Type   | Description                                     |
+|------------------------------------------|--------|-------------------------------------------------|
+| `os.name`       | string | The OS name as reported by the by the device (User-Agent HTTP header)       |
+| `os.version`  | string | The OS version as reported by the by the device (User-Agent HTTP header)  |
+| `os.version_major`   | string | The OS version major as reported by the by the device (User-Agent HTTP header)   |
+
+### Geo-location
+
+The following attributes are related to the geo-location of IP addresses:
+
+| Fullname                                    | Type   | Description                                                                                                                          |
+|:--------------------------------------------|:-------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| `geo.country`         | string | Name of the country                                                                                                                  |
+| `geo.country_iso_code`     | string | [ISO Code][7] of the country (for example, `US` for the United States, `FR` for France).                                                  |
+| `geo.country_subdivision`     | string | Name of the first subdivision level of the country (for example, `California` in the United States or the `Sarthe` department in France). |
+| `geo.country_subdivision_iso_code` | string | [ISO Code][7] of the first subdivision level of the country (for example, `CA` in the United States or the `SA` department in France).    |
+| `geo.continent_code`       | string | ISO code of the continent (`EU`, `AS`, `NA`, `AF`, `AN`, `SA`, `OC`).                                                                 |
+| `geo.continent`       | string | Name of the continent (`Europe`, `Australia`, `North America`, `Africa`, `Antartica`, `South America`, `Oceania`).                    |
+| `geo.city`            | string | The name of the city (example `Paris`, `New York`).                                                                                   |
+## User attributes
+
+In addition to default attributes, add user related data by [identifying user sessions][8]. This lets you follow the journey of a given user, figure out which users are the most impacted by errors and monitor performance for your most important users.
```

Comment:
This link is pointing to "add global context" in the advanced config page. Should it rather point to `/real_user_monitoring/browser/advanced_configuration/?tab=npm#identify-user-sessions`?

Reply:
Indeed, good catch!